src/Driver/DriverInterface.php

```diff
@@ -1,11 +1,17 @@
 <?php
 
-/*
+/**
  * This file is part of the Mink package.
  * (c) Konstantin Kudryashov <ever.zet@gmail.com>
  *
  * For the full copyright and license information, please view the LICENSE
  * file that was distributed with this source code.
+ *
+ * @noinspection PhpLanguageLevelInspection
+ * @noinspection PhpMissingParamTypeInspection
+ * @noinspection PhpMissingReturnTypeInspection
+ * @noinspection ReturnTypeCanBeDeclaredInspection
+ * @noinspection PhpElementIsNotAvailableInCurrentPhpVersionInspection
```

Comment:
I've mixed feeling if this should be here: Disabling these inspections makes it nicer for the developer to focus on the bigger problems, but it's also basically hiding some problems.